__tests__/integration/utils/test-utils-integration.js

```diff
@@ -181,16 +200,85 @@ export class TestUtils {
     return response.body.address;
   }
 
+  /**
```

Comment:
**Comment unrelated to line number**

Maybe we should create an issue to move all those methods from the `TestUtils` that communicate with the headless API to a new library, maybe a `headless-client` library

What do you think?

Reply:
Seems like a great idea! Opened issue #162 for this discussion.